development/prow-installer/cmd/installer/main.go

```diff
@@ -134,23 +139,23 @@ func main() {
 			saname = fmt.Sprintf("%.30s", saname)
 			_, err = crmClient.AddSAtoRole(saname, serviceAccount.Roles, readConfig.Project, nil)
 			if err != nil {
-				log.Errorf("Failed assign sa %s to roles, got: %w", saname, err)
+				log.Errorf("Failed assign sa %s to roles, got: %v", saname, err)
 			}
 		}
 	}
 
 	for k, v := range readConfig.Clusters {
 		clusterID := fmt.Sprintf("%s-%s", readConfig.Prefix, v.Name)
-		k8sclient, err = k8s.NewClient(ctx, clusterID, v.Location, gkeClient)
+		k8sclient, kubectlWrapper, err := k8s.NewClient(ctx, clusterID, v.Location, gkeClient)
 		if err != nil {
 			log.Fatalf("failed get k8s client, got: %v", err)
 		}
 		v.K8sClient = k8sclient
+		v.KubectlWrapper = kubectlWrapper
 		populator := k8s.NewPopulator(k8sclient)
 		v.Populator = populator
 		readConfig.Clusters[k] = v
```

Comment:
I don't know if this is a safe operation, modifying the Clusters array while looping through it. Do we run our tests with `-race` enabled?